WalletWasabi.Gui/WalletWasabi.Gui.csproj

```diff
@@ -64,6 +64,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Avalonia.Native" Version="0.8.2" />
```

Comment:
This is still not resolved.

Reply:
@danwalmsley can we just update AvalonStudio.Shell nad not referencing new Package?

Reply:
AvalonStudio.Shell references these library as well, so basically there is no new reference.
 
With Avalonia 0.9 we will need to do the same thing. Both Shell and Avalonia will be referenced.

Reply:
That's not what we agreed on with @danwalmsley. We use everything Avalonia through the shell.

Reply:
Solved!